lib/multicell_location/services/nrf_cloud_integration.c

```diff
@@ -98,14 +101,15 @@ int location_service_get_cell_location_nrf_cloud(
 		LOG_ERR("Cloud connection is not established");
 		return err;
 	} else if (err) {
-		LOG_ERR("Failed to request cell pos data, error: %d", err);
+		LOG_ERR("Failed to request cellular positioning data, error: %d", err);
 		return err;
 	}
 
 	LOG_INF("Cellular positioning request sent");
 
 	if (k_sem_take(&location_ready, K_SECONDS(20)) == -EAGAIN) {
-		LOG_ERR("Cellular positioning data request timed out");
+		LOG_ERR("Cellular positioning data request timed out or "
+			"cloud returned an invalid location");
```

Comment:
**nitpick** Would something like "cloud did not return a location" be more appropriate than "cloud returned an invalid location"? Because cloud does not return a location in this case, right?

Reply:
yes, it's better